manifests/integreatly-rhsso/10.0.0/keycloak-operator.v10.0.0.clusterserviceversion.yaml

```diff
@@ -182,6 +182,8 @@ spec:
                   - command:
                       - keycloak-operator
                     env:
+                      - name: RELATED_IMAGE_RHSSO_OPENJDK
+                        value: registry.redhat.io/rh-sso-7/sso74-openshift-rhel8:7.4-3
```

Comment:
How about using the floating tag - `7.4`? This way you won't need to bother about upgrading the image to the latest patch. All micro updates should be backwards compatible.

Reply:
There was discussion around this and we decided to use the specific tag rather then floating as we have expereinced problems with floating tags in the past

Reply:
Ok, thanks for the explanation.